### DISCUSSION TITLE
No data available in generated PCAPs

### DISCUSSION THREAD

**Discussion by madcaban** [Security-Onion-Solutions/securityonion#8143] [Unsupported Versions] [answered]
Hello, 

I have a problem in version 2.3.120 that when trying to get a PCAP from an alert/event, it is generated but returns the error "no data available". It happens both when generating it from an alert/event and from the PCAP section.

It should be noted that this case only occurs with one of the sensor nodes (deployment mode) where the manager is located in MV, the affected probe is located in AWS and the rest of the probes are established in hardware. The rest of the minions are working correctly and it is possible to obtain the PCAPs.

I have checked that the mirroring is working correctly and the sensor receives correctly the traffic (full packet capture) and also the alerts/events are detected as expected.

Is this a configuration error or a version error?

Thanks for your help and best regards.
David

**Comment by weslambert** [comment, marked as answer]
It's more so a characteristic of how the traffic is handled when pulling PCAP for VXLAN encapsulated traffic. Traffic in AWS is encapsulated using a VXLAN tunnel, so we have to handle the traffic and walk back connections a little bit differently. This used to work pretty well, but it seems like maybe some changes have decreased reliability. We are working on improving this for a future release, but for the time being, you could do something like the following (assuming you are using Zeek for metadata):

- From the alert, click Correlate to correlate with other records
- From the associated `conn` record in the results, search for the tunnel parent record, which should be of type `vxlan`
- From the `vxlan` record (which should be the parent tunnel record for the decapsulated `conn` record), attempt to pull the PCAP

**Reply by madcaban** [reply]
Hi weslambert,

Thank you so much for your reply. The workaround you provide works, but it is difficult to perform every time a PCAP is needed.

Do you have any clues on when this feature will be released? It is an interesting functionality for my enviroment.

Thank you very much for your help,
David

**Reply by weslambert** [reply]
We are currently tracking improvement of this behavior here: https://github.com/Security-Onion-Solutions/securityonion/issues/7985.

Unfortunately, I can't provide a date for when it will be included with a release.
